### PR TITLE
Add Slack to README

### PR DESCRIPTION
**Pull request by casassg**
Add Slack link to Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![TFX Addons package CI](https://github.com/tensorflow/tfx-addons/workflows/TFX%20Addons%20package%20CI/badge.svg?branch=main)](https://github.com/tensorflow/tfx-addons/actions?query=branch%3Amain)
 
 ## TL;DR
-This is the repo for projects organized under the special interest group, SIG TFX-Addons. Join the group by [joining the Google Group](http://goo.gle/tfx-addons-group) and participate in projects to build new components, libraries, tools, and other useful additions to TFX.
+This is the repo for projects organized under the special interest group, SIG TFX-Addons. Join the group by [joining the Google Group](http://goo.gle/tfx-addons-group) or our [Slack](https://join.slack.com/t/tfxaddons/shared_invite/zt-tu1981lj-npIhRSHF8gl9G0ldUovbcw) and participate in projects to build new components, libraries, tools, and other useful additions to TFX.
```
Comment (rcrowe-google) [inline README.md:6]: I'd prefer to word this differently.  We want everyone to join the Google Group, and the Slack channel is an optional extra.

**Review by rcrowe-google — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (casassg) [reply README.md]: Moved down :D

**Review by hanneshapke — APPROVED** [review]

**Comment by casassg** [comment]
@rcrowe-google let's merge this one!

**Review by rcrowe-google — APPROVED** [review]
Thanks!

**Pull request merged**